src/main/java/com/royalrangers/bean/achievement/UserRewardBean.java

```diff
@@ -0,0 +1,17 @@
+package com.royalrangers.bean.achievement;
+
+import com.royalrangers.model.achievement.Reward;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class UserRewardBean {
+    private Long id;
+    private Date createDate;
+    private Date updateDate;
```

Comment:
This bean used for view data in response methods GET. CreateDate and UpdateDate returned for view on a client.